### DISCUSSION TITLE
Osu! Lazer Android Chromebook Drawing Tablet Glitch

### DISCUSSION THREAD

**Discussion by Sambamc137** [ppy/osu#22701] [Q&A]
### Type

Game behaviour

### Bug description

 I connected my drawing tablet to my chromebook in osu lazer android and when you put the stylus anywhere near the edge of the screen the cursor glitches and teleports to another random point on the screen.

### Version

2023.207.0 lazer

### Logs

[database.log](https://github.com/ppy/osu/files/10778497/database.log)
[input.log](https://github.com/ppy/osu/files/10778498/input.log)
[network.log](https://github.com/ppy/osu/files/10778499/network.log)
[performance.log](https://github.com/ppy/osu/files/10778500/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10778501/runtime.log)

**Comment by bdach** [comment]
Does toggling the "tablet" input handler in settings or disabling high precision mouse fix it?

**Reply by Sambamc137** [reply]
There's no tablet input option in android osu lazer for me at least, and disabling high precision mouse makes it so the game doesn't even recognize there's a mouse so it makes it so the system mouse is visible and movable but the game doesn't recognize it.
